.github/workflows/build.yml

```diff
@@ -2,7 +2,7 @@ name: Generate Dashboard
 on:
   workflow_dispatch:
   schedule:
-    - cron: "0 * * * *"  # Runs every hour
+    - cron: "0 5 * * *"  # Runs at 05:00 every day
```

Comment:
I think a higher frequency was requested by ecdc but I think every hour is overkill.
If we need an update, we can always run the workflow manually with the dispatch trigger.